concordium-sdk/src/main/java/com/concordium/sdk/transactions/BlockItem2.java

```diff
@@ -0,0 +1,28 @@
+package com.concordium.sdk.transactions;
+
+import com.concordium.sdk.transactions.account.IAccountTransaction2;
+import com.concordium.sdk.transactions.account.IAccountTransactionPayload2;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+
+import java.nio.ByteBuffer;
+
+@RequiredArgsConstructor
+public abstract class BlockItem2<P extends IAccountTransactionPayload2> {
+    private final TransactionVersion2 version;
+    private final IAccountTransaction2<P> transaction;
+
+    public byte[] serialize() {
```

Comment:
**Should we prefer names like `serialize` over `getBytes` ?**
As `getBytes` does not relay any information about how they are serialised. While serialize method with `byte[]` return type shows that the bytes are serialised in some particular protocol.